docs/user/hosting.rst

```diff
@@ -1,28 +1,71 @@
-Documentation Hosting Features
-==============================
+.. old label
+.. _Documentation Hosting Features:
 
-The main way that users interact with your documentation is via the hosted HTML that we serve.
-We support a number of important features that you would expect for a documentation host.
+Hosting features provided by Read the Docs
+==========================================
 
-.. contents:: Contents
-   :local:
-   :depth: 1
+This article provides a quick overview and reference to how documentation is *hosted* on Read the Docs.
+Hosting is optimized with documentation in mind and a number of special features are provided,
+some of which may be customized.
+
+.. seealso::
+
+   :doc:`/reference/features`
+     For a full list of Read the Docs features, see the features reference page.
+
+Overview of hosting features
+----------------------------
 
 Subdomain support
------------------
+  Every project has a subdomain that is available to serve its documentation based on it's :term:`slug`.
+  If you go to ``<slug>.readthedocs.io``, it should show you the latest version of your documentation,
+  for example https://docs.readthedocs.io.
+  For :doc:`/commercial/index` the subdomain looks like ``<slug>.readthedocs-hosted.com``.
 
-Every project has a subdomain that is available to serve its documentation based on it's :term:`slug`.
-If you go to ``<slug>.readthedocs.io``, it should show you the latest version of your documentation,
-for example https://docs.readthedocs.io.
-For :doc:`/commercial/index` the subdomain looks like ``<slug>.readthedocs-hosted.com``.
+Custom domain support
+  Documentation projects may also define their own domains.
+  A project may define multiple domains,
+  as well as define which is the *canonical domain*.
 
-.. seealso::
+  ⏩️ :doc:`Read more </custom-domains>`
+
+Multiple documentation versions
+  The built-in URL structure of Read the Docs is pre-defined to allow for multiple versions and translations to be hosted.
+  You can also switch off this mechanism with :doc:`Single-Version projects </single_version>`.
```

Comment:
We should never say `mechanism` -- but more importantly, we should explain the benefit rather than saying you can disable it..

```suggestion
  If you only have 1 version and translation,
  we support :doc:`single version projects </single_version>`.
```